packages/orchestrator-ui-components/src/utils/getToastMessage.ts

```diff
@@ -0,0 +1,35 @@
+import { ToastTypes } from '../types';
+import type { Toast } from '../types';
+
+export const getToastMessage = (
+    type: ToastTypes,
+    text: string, // We use string here instead of Toast['text'] because we want to prevent passing in react component because they trigger an "unsynchronizable values in payload detected" error',
```

Comment:
Is that because the redux store cannot store the toast otherwise?

Reply:
I've only moved this code around so this was already there. I don't exactly remember but I think it was that redux does store it but you loose the ability to a do history playback in the redux tools which is nice when debugging.